src/core_ocean/shared/mpas_ocn_vmix.F

```diff
@@ -1078,6 +1221,13 @@ subroutine ocn_vmix_implicit(dt, meshPool, diagnosticsPool, statePool, forcingPo
             dt, kineticEnergyCell, &
             vertViscTopOfEdge, layerThickness, layerThicknessEdge, normalVelocity, &
             ssh, bottomDepth, err)
+        else if (config_use_implicit_bottom_roughness) then
```

Comment:
May want to be clear here that this is `config_use_implicit_bottom_roughness_z0` to make it crystal clear.